lessons/Data-Science/measures-of-center/langs/en-us/index.adoc

```diff
@@ -346,54 +249,83 @@ Here are just a few examples of summary data being used for important things:
+}
+
+@teacher{Question 3 requires students to _apply_ their knowledge of mean *and* median, which can be quite difficult. Commonly, students' understanding of center does not extend beyond algorithms. Invite students to think back to what they know about histograms and histogram shape. Challenge them to think deeply about how a histogram's shape relates to its measures of center. We will continue to consider this topic in the next lesson section.}
+
+
 @slidebreak
 
+
+Let's summarize some of the key ideas we encountered while thinking about the best measure of center to summarize the pounds column of the animals dataset.
+
 @QandA{
-@Q{Given the extreme income inequality in the United States, what measure of center would best represent a typical family income?}
-@A{The median}
+
+@Q{When is mean probably the best measure of center to use?}
+@A{The @vocab{mean} is a useful summary number when all of the points in a dataset are fairly balanced on either side of the middle.}
+
+
+@Q{Although mean is generally the best measure of center, statisticians sometimes fall back to the median. When is median the best measure of center to use?}
+@A{For skewed datasets, the @vocab{median} is a better summary value because it is less sensitive to skew. Mean is misleading for datasets with imbalance and extreme outliers.}
+
+@Q{In what situations is mode the best measure of center?}
+@A{The @vocab{mode} is a useful measure of center when we have a dataset with a small number of values. Mode is also our only measure of center that can be used with categorical data.}
 }
 
 @slidebreak
-Consider how many policies or laws are informed by statistics like this! Knowing about measures of center helps us see through misleading statements.
 
+Consider how many policies or laws are informed by statistics! Knowing about measures of center helps us see through and critique misleading statements.
 
-*Here are some guidelines for when to use which measure of center:*
-
-- If the data doesn’t show much skewness or have outliers, @vocab{mean} is the best summary because it incorporates information from every value.
-- If the data has noticeable outliers or skewness, @vocab{median} gives a better summary of center than the mean.
-- If there are very few possible values, such as AP Scores (1–5), @vocab{mode(s)} could be a useful way to summarize the dataset.
 
 @slidebreak
 
 @lesson-instruction{
-- Choose a column from the Animals dataset and complete the second half of @printable-exercise{summarizing-columns-moc.adoc}. As you work, think about what the measures of center tell you about the shape of the dataset.
-- Then complete @printable-exercise{critiquing-findings.adoc}. (You will be computing these measures of center without @proglang.)
-- Practice the Data Cycle with measures of center, using @printable-exercise{data-cycle-practice.adoc}. 
+- Use @proglang to complete @printable-exercise{critiquing-findings.adoc}.
```

Comment:
Wondering if we should rename this page, to bring it in line with the Fake News page?

Reply:
hm, the "Fake News" stuff hasn't been introduced yet in the pathway. If we were to change the name of the page, I would want to highlight that we have a whole lesson on ways that data can be misused. You think we should do that?